test/controllers/SeriesDetailsControllerSpec.scala

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import org.scalatest.Matchers._
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import util.FrontEndTestHelper
+
+class SeriesDetailsControllerSpec extends FrontEndTestHelper {
+  "SeriesDetailsController GET" should {
+
+    "render the series details page with an authenticated user" in {
+      val controller = new SeriesDetailsController(getAuthorisedSecurityComponents())
+      val home = controller.seriesDetails().apply(FakeRequest(GET, "/"))
```

Comment:
Maybe I'm misunderstanding how these controller tests work, but why is this variable called `home`? And why do you make a request to "/" (which looks like the homepage route) to the SeriesDetailsController?

Reply:
Resolved